js/views/Settings.js

```diff
@@ -0,0 +1,43 @@
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+
+export default class Settings extends View {
```

Comment:
You don't need to give the class a name, since you're not referencing it anywhere. So, it could be:

```
export default class extends View {
```